Pipfile

```diff
@@ -11,7 +11,7 @@ indexclient = {editable = true, path = "."}
 pytest = "*"
 mock = "*"
 cdisutilstest = {editable = true, git = "https://github.com/uc-cdis/cdisutils-test.git", ref = "1.0.0"}
-indexd = {git = "https://github.com/uc-cdis/indexd.git", ref = "4.0.2"}
+indexd = {git = "https://github.com/uc-cdis/indexd.git", ref = "master"}
```

Comment:
Yeah, I'm not sure how to handle the circular dependency here. I went with master since I assumed any changes to indexclient would be for the latest indexd, so a developer would want the unit tests in indexclient to hit master indexd. 

If that's a poor assumption, then I can revert it but thats puts us back in the previous situation where the tests in indexclient won't pass until we release indexd...which is currently depending on an unreleased indexclient...which is currently depending on an unreleased indexd (@_@).